packages/dd-trace/src/config.js

```diff
@@ -782,21 +393,161 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
+    this._setValue(defaults, 'dbmPropagationMode', 'disabled')
+    this._setBoolean(defaults, 'dsmEnabled', false)
+    this._setBoolean(defaults, 'openAiLogsEnabled', false)
+    this._setValue(defaults, 'url', undefined)
+    this._setValue(defaults, 'site', 'datadoghq.com')
+    this._setValue(defaults, 'hostname', '127.0.0.1')
+    this._setValue(defaults, 'port', '8126')
+    // this._setValue(defaults, 'flushInterval', defaultFlushInterval)
+    this._setValue(defaults, 'flushMinSpans', 1000)
+    this._setValue(defaults, 'queryStringObfuscation', qsRegex)
+    this._setBoolean(defaults, 'clientIpEnabled', false)
+    this._setValue(defaults, 'clientIpHeader', null)
+    this._setBoolean(defaults, 'plugins', true)
+    this._setValue(defaults, 'dogstatsd.hostname', '127.0.0.1')
+    this._setValue(defaults, 'dogstatsd.port', '8125')
+    this._setBoolean(defaults, 'runtimeMetrics', false)
+    this._setBoolean(defaults, 'experimental.runtimeId', false)
+    this._setValue(defaults, 'experimental.exporter', undefined)
+    this._setBoolean(defaults, 'experimental.enableGetRumData', false)
+    this._setValue(defaults, 'sampler.rateLimit', undefined)
+    this._setBoolean(defaults, 'reportHostname', false)
+    this._setValue(defaults, 'scope', undefined)
+    this._setBoolean(defaults, 'profiling.enabled', false)
+    this._setBoolean(defaults, 'profiling.sourceMap', true)
+    this._setValue(defaults, 'profiling.exporters', 'agent')
+    this._setValue(defaults, 'spanAttributeSchema', 'v0')
+    this._setValue(defaults, 'spanRemoveIntegrationFromService', false)
+    this._setValue(defaults, 'peerServiceMapping', {})
+    this._setValue(defaults, 'lookup', undefined)
+    this._setBoolean(defaults, 'startupLogs', false)
+    this._setBoolean(defaults, 'telemetry.enabled', true)
+    this._setBoolean(defaults, 'telemetry.logCollection', false)
+    this._setValue(defaults, 'telemetry.heartbeatInterval', 60000)
+    this._setBoolean(defaults, 'telemetry.debug', false)
+    this._setBoolean(defaults, 'telemetry.metrics', true)
+    this._setBoolean(defaults, 'telemetry.dependencyCollection', true)
+    this._setValue(defaults, 'protocolVersion', '0.4')
+    this._setValue(defaults, 'tagsHeaderMaxLength', 512)
+    this._setValue(defaults, 'spanComputePeerService', false)
+    this._setBoolean(defaults, 'appsec.enabled', undefined)
+    this._setValue(defaults, 'appsec.rules', undefined)
+    this._setValue(defaults, 'appsec.customRulesProvided', false)
+    this._setValue(defaults, 'appsec.rateLimit', 100)
+    this._setValue(defaults, 'appsec.wafTimeout', 5e3)
+    this._setValue(defaults, 'appsec.obfuscatorKeyRegex', defaultWafObfuscatorKeyRegex)
+    this._setValue(defaults, 'appsec.obfuscatorValueRegex', defaultWafObfuscatorValueRegex)
+    this._setValue(defaults, 'appsec.blockedTemplateHtml', undefined)
+    this._setValue(defaults, 'appsec.blockedTemplateJson', undefined)
+    this._setBoolean(defaults, 'remoteConfig.enabled', true)
```

Comment:
the default depends on if we're in a serverless env or not

Reply:
when none of the env vars are set, [`inServerlessEnvironment `](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/config.js#L210) is false, so [`remoteConfig.enabled`](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/config.js#L434) would be true. I'm assuming by default all env vars are undefined, is that right?